client/sections/services/sections.service.coffee

```diff
@@ -1,8 +1,11 @@
-Sections = new Mongo.Collection('sections')
-
 angular.module 'sections'
 .service 'sectionsServ', ($meteor) ->
     all: $meteor.collection(Sections).subscribe('sections')
 
     getTopics: (section_id) ->
```

Comment:
Tam też powinno być zmienione według mnie. Jak masz na przykład takie railsy, to tam masz controller, który odpowiada za dany model, nie ważne, czy akcja index pobiera wszystkie czy wyfiltrowane dane.